serializers/postgres.rb

```diff
@@ -27,7 +27,10 @@ def self.serialize_internal(pg, options = {})
       )
 
       if pg.timeline && pg.representative_server&.primary?
-        base[:earliest_restore_time] = pg.timeline.earliest_restore_time&.utc&.iso8601
+        begin
+          base[:earliest_restore_time] = pg.timeline.earliest_restore_time&.utc&.iso8601
+        rescue
```

Comment:
Should we log the exception to figure out the cause in the future?

Reply:
makes sense. added logging.